src/experimental/tools/VideoThumbnailLoader/video_thumbnail_loader_error.ts

```diff
@@ -25,7 +25,7 @@ export default class VideoThumbnailLoaderError extends Error {
    * @param {string} message
    */
   constructor(code: string, message: string) {
-    super();
+    super(message);
```

Comment:
Strange that we don't have the usual `CODE: message` here (I know that was already the case, I'm just wondering why)

Reply:
oh I didn't notice when replacing, I have added it, I guess it was forgotten